pipeline/tasks/protoc_tasks.py

```diff
@@ -218,8 +259,8 @@ def execute(self, src_proto_path, import_proto_path, output_dir,
         #   - it doesn't have to
         #   - and multiple invocation will overwrite the desc_out_file
         self.exec_command(
-            _protoc_header_params(
-                header_proto_path, toolkit_path) +
+            ['protoc'] +
```

Comment:
Since this task is being run in the gapic config pipeline having a proto params object doesn't make much sense since the proto params object is retrieved based on the language.